Solutions/Branwen/GUI.cs

```diff
@@ -9,324 +9,460 @@
+						SheetData sheetData = worksheetPart.Worksheet.GetFirstChild<SheetData>();
+						Sheet sheet = new Sheet();
+						sheet.Id = spreadsheetDocument.WorkbookPart.GetIdOfPart(worksheetPart);
+						sheet.Name = topLevelDirectories[i].Name;
+						sheet.SheetId = (UInt32)Convert.ToInt32(i + 1);
+						sheets.Append(sheet);
+						WriteDirectoryToWorksheet(RunInventory(topLevelDirectories[i], MediaDriveNumberTextBox.Text), sheetData);
+					}
+					spreadsheetDocument.Close();
+
+					#endregion
+				}
+				MessageBox.Show("DONE! Files Inventoried:  " + fileCount);
+			}
+			catch (Exception ex)
+			{
+				MessageBox.Show("Failed to Inventory:" + Environment.NewLine + ex.Message);
+			}
+			SelectAndRunInventoryButton.Enabled = true;
+			SelectAndRunInventoryButton.Text = "Select Inventory Directory";
+			UseDBCheckBox.Enabled = true;
+			SetEnabledControls();
+		}
+
+		/// <summary>
+		/// Truncates the Database to clear all items
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="e"></param>
+		private void WipeDbButton_Click(object sender, EventArgs e)
+		{
+			SelectAndRunInventoryButton.Enabled = false;
+			SelectAndRunInventoryButton.Text = "Working";
+			WipeDbButton.Enabled = false;
+			WipeDbButton.Text = "...";
+			MediaDriveNumberTextBox.Enabled = false;
+			UseDBCheckBox.Enabled = false;
+			ExportFileCheckBox.Enabled = false;
+
+			try
+			{
+				MySqlConnection mySqlConnection;
+				mySqlConnection = new MySqlConnection("server=box654.bluehost.com;user=lbkstud1_smedia;password=#sucK_my_d1ck;database=lbkstud1_SaurutobiMedia;");
+				mySqlConnection.Open();
+
+				string deleteStatement = "TRUNCATE TABLE SaurutobiMediaPaths; TRUNCATE TABLE SaurutobiMediaFiles;";
+				MySqlCommand deleteCommand = new MySqlCommand(deleteStatement, mySqlConnection);
+				deleteCommand.ExecuteNonQuery();
+			}
+			catch (Exception ex)
+			{
+				MessageBox.Show("Failed to Wipe:" + Environment.NewLine + ex.Message);
+			}
+			SelectAndRunInventoryButton.Enabled = true;
+			SelectAndRunInventoryButton.Text = "Select Inventory Directory";
+			UseDBCheckBox.Enabled = true;
+			SetEnabledControls();
+		}
+
+		/// <summary>
+		/// Switches DB-related controls Enabled property based on if it's checked
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="e"></param>
+		private void UseDBCheckBox_CheckedChanged(object sender, EventArgs e)
+		{
+			SetEnabledControls();
+		}
+
+		#endregion
+
+		#region Database Methods
+
+		/// <summary>
+		/// Writes all files in the Directory to the Databbase
+		/// </summary>
+		/// <param name="files"></param>
+		private void WriteDirectoryToDatabase(IEnumerable<BranwenFileInfo> files, MySqlConnection mySqlConnection)
+		{
+			string insertFilesStatement = "INSERT INTO SaurutobiMediaFiles (FileName, Extension, Size, MediaType, MediaDrive) VALUES ";
+			string insertPathsStatement = "INSERT INTO SaurutobiMediaPaths (FileName, PartOfPath, Path, MediaDrive) VALUES ";

```

Comment:
I might suggest using StringBuilder so that the program doesn't destroy/create strings at runtime tons of times in your loop.  StringBuilder optimizes that for you and doesn't do the whole destroy/create stuff when you want to append strings.